pkg/backends/newrelic/newrelic.go

```diff
@@ -307,13 +317,24 @@ func (n *Client) post(ctx context.Context, buffer *bytes.Buffer, data interface{
 	b.Clock = clck
 	b.Reset()
 	b.MaxElapsedTime = n.maxRequestElapsedTime
+
+	var next time.Duration
+	var retryAfterErr *RetryAfterError
```

Comment:
Purely as a stylistic thing, `next` can be a `:=`, and `retryAfterErr` should be declared where it's consumed.  No need to change it though, just a consideration.